django/urls/resolvers.py

```diff
@@ -71,6 +72,27 @@ def __repr__(self):
             )
         )
 
+    def __getstate__(self):
+        state = self.__dict__.copy()
+
+        if self.is_class_based:
+            # The following attributes of ResolverMatch prevent pickling.
+            del state['func']
+            del state['tried']
```

Comment:
`tried` can be also an issue for function-based views because `URLResolver` is not pickleable, everything depends on the order in `urlpatterns`, see e.g.
```diff
diff --git a/tests/urlpatterns/path_urls.py b/tests/urlpatterns/path_urls.py
index 29b6185292..0efb713959 100644
--- a/tests/urlpatterns/path_urls.py
+++ b/tests/urlpatterns/path_urls.py
@@ -3,13 +3,13 @@ from django.urls import include, path, re_path
 from . import views
 
 urlpatterns = [
+    path('included_urls/', include('urlpatterns.included_urls')),
     path('articles/2003/', views.empty_view, name='articles-2003'),
     path('articles/<int:year>/', views.empty_view, name='articles-year'),
     path('articles/<int:year>/<int:month>/', views.empty_view, name='articles-year-month'),
     path('articles/<int:year>/<int:month>/<int:day>/', views.empty_view, name='articles-year-month-day'),
     path('users/', views.empty_view, name='users'),
     path('users/<id>/', views.empty_view, name='user-with-id'),
-    path('included_urls/', include('urlpatterns.included_urls')),
     re_path(r'^regex/(?P<pk>[0-9]+)/$', views.empty_view, name='regex'),
     re_path(r'^regex_optional/(?P<arg1>\d+)/(?:(?P<arg2>\d+)/)?', views.empty_view, name='regex_optional'),
     re_path(
```